usaspending_api/api_contracts/scripts/mock/drakov.js

```diff
@@ -4,7 +4,7 @@ const drakovOptions  = {
     sourceFiles: './contracts/**/*.md',
     serverPort: 5000,
     autoOptions: true,
-    delay: 10000 // 10 seconds
+    delay: 500 // 1/2 a second
```

Comment:
Agree that we should make this a shorter delay - maybe more like 1-2s for previewing loading states?

Reply:
@ebdabbs -- agreed!